test/e2e/query_test.go

```diff
@@ -467,7 +527,7 @@ func queryAndAssert(t *testing.T, ctx context.Context, addr string, q string, op
 	t.Helper()
 
 	sortResults(expected)
-	result := instantQuery(t, ctx, addr, q, opts, len(expected))
+	result := instantQuery(t, ctx, addr, q, opts, len(expected), "")
```

Comment:
can you add in a constant for empty string? Hard to read and understand what is `""` :)